vmdb/app/views/shared/summary/_textual_multilabel.html.haml

```diff
@@ -11,5 +11,9 @@
     - items[:values].each do |item|
       %tr
         - item.each do |value|
-          %td
-            = value
+          - if value.kind_of?(Hash)
+            %td{:title => value[:title]}
```

Comment:
+1 for a resizing/popup div, having large tooltips is not helping (copy/paste etc.)